### ISSUE TITLE
Lambda functions nonexistent after deletion and re-apply

### ISSUE THREAD

**Issue by JohnLZeller** [jorgebastida/gordon#48]
I ran `gordon apply` to publish my lambdas, which worked fine. Then, I deleted those Lambdas manually. Now, when I attempt to run `gorgon apply`, it says that there is nothing to change, when really those Lambdas are not there any more.

Perhaps I am missing something but it seems like this should republish.

**Comment by spullara** [comment]
I believe that it only looks locally to see if you changed anything.

**Comment by jorgebastida** [comment]
Hello @JohnLZeller, yes that is the case.

Gordon uses CloudFormation underneath for deploying resources into AWS. 
When you use CloudFormation for managing resources, you are not suppose to manage those manually... not to mention delete them. This is a well known behaviour, and the expected result.

tl;dr Resources deployed using gordon, should be managed using gordon.
